packages/showcases/react/stories/components/structure/VtmnDivider/VtmnDivider.stories.tsx

```diff
@@ -25,7 +25,7 @@ export default {
 
 const Template: Story = (args) => (
   <div style={{ width: 400, height: 250 }}>
-    <VtmnDivider {...args} />
+    <VtmnDivider aria-labelledby="title-example" {...args} />
```

Comment:
And where is the element with the id `#title-example`? If you add this, you need to add this element in order to label the divider.